transport/grpc/dial.go

```diff
@@ -144,7 +144,7 @@ func dial(ctx context.Context, insecure bool, o *internal.DialSettings) (*grpc.C
 			}
 			grpcOpts = []grpc.DialOption{
 				grpc.WithCredentialsBundle(
-					grpcgoogle.NewComputeEngineCredentials(),
+					grpcgoogle.NewComputeEngineCredentials(oauth.TokenSource{creds.TokenSource}),
```

Comment:
This code does not compile, this method does not take any arguments.

Reply:
Yes we will need to wait for grpc/grpc-go#4767. You may want to add a `do not merge` label for now.